tests/test_doctestplus.py

```diff
@@ -651,3 +651,35 @@ def test_doctest_float_replacement(tmpdir):
 
     doctest.testfile(str(test2_rst), module_relative=False,
                      raise_on_error=True, verbose=False, encoding='utf-8')
+
+
+def test_doctest_subpackage_requires(testdir):
+    testdir.makeini(
+        """
+        [pytest]
+        doctest_subpackage_requires =
+            test/a/*:pytest>1
+            test/b/*:pytest>1;averyfakepackage>99999.9
+            test/c/*:anotherfakepackage>=22000.1.2
+    """
+    )
+    test = testdir.mkdir('test')
+    a = test.mkdir('a')
+    b = test.mkdir('b')
+    c = test.mkdir('c')
+
+    pyfile = dedent("""
+        def f():
+            '''
+            >>> 1
+            1
+            '''
+            pass
+    """)
+
+    a.join('testcode.py').write(pyfile)
+    b.join('testcode.py').write(pyfile)
+    c.join('testcode.py').write(pyfile)
+
+    reprec = testdir.inline_run(test, "--doctest-plus")
+    reprec.assertoutcome(passed=1)
```

Comment:
Is there a way to check that the one passed is actually "a"?

Reply:
Done